ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import CoreGraphics
+
+/// Represents a two-part shadow as used by FluentUI.
+public struct ShadowInfo {
+    let colorOne: DynamicColor
+    let blurOne: CGFloat
+    let xOne: CGFloat
+    let yOne: CGFloat
+    let colorTwo: DynamicColor
+    let blurTwo: CGFloat
+    let xTwo: CGFloat
+    let yTwo: CGFloat
```

Comment:
Any reason for the switch from 1 to One?

Reply:
I generally try to avoid numerals in variable or method names wherever possible, though it can switch back if you'd prefer?

Reply:
I don't have a strong preference, just curios.